travis/python_tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+status=0
+
+function run_test {
```

Comment:
This script in xPro echoes the command before running it. Very useful for reading the test output. Can you add that?

https://github.com/mitodl/mitxpro/blob/master/travis/python_tests.sh#L4-L17